shared/src/main/java/gov/hhs/cdc/trustedintermediary/ruleengine/Rule.java

```diff
@@ -1,19 +1,17 @@
-package gov.hhs.cdc.trustedintermediary.etor.ruleengine;
+package gov.hhs.cdc.trustedintermediary.ruleengine;
 
 import gov.hhs.cdc.trustedintermediary.context.ApplicationContext;
-import gov.hhs.cdc.trustedintermediary.wrappers.HapiFhir;
+import gov.hhs.cdc.trustedintermediary.wrappers.HealthData;
+import gov.hhs.cdc.trustedintermediary.wrappers.HealthDataExpressionEvaluator;
 import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
 import java.util.List;
 
-/**
- * Represents a rule that can be run on a FHIR resource. Each rule has a name, description, logging
- * message, conditions to determine if the rule should run, and actions to run in case the condition
- * is met.
- */
```

Comment:
Looks like we nuked the JavaDocs.  Let's get a new JavaDoc added.  Or the original one if it still applies.